packages/syft/src/syft/grid/client/grid_connection.py

```diff
@@ -39,7 +38,8 @@ class GridHTTPConnection(HTTPConnection):
 
```

Comment:
Could we use `@serializable(recursive_serde=True)` and remove the proto file and manual serde steps?

Reply:
Sure , I will change it